wrappers/src/utf8.hpp

```diff
@@ -0,0 +1,380 @@
+/*************************************************************************
```

Comment:
What is this doing?

Reply:
This was a file that does utf8<->utf16 conversion - it was previously in Core but got deleted.